### ISSUE TITLE
failed to apply provision: not enough free resources to support this memory size

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/itenv_testnet#8]
![image](https://user-images.githubusercontent.com/64129/117670782-7cb83080-b1a8-11eb-9daf-b0d66a91ede7.png)

as per our call earlier today this error shouldn't happen in case of new deployments.

**Comment by muhamadazmy** [comment]
No, you got it wrong, you will now get "no free memory to support this resource" more often than before because the node will reject any workload that can cause memory over provision.

**Comment by sasha-astiadi** [comment]
@xmonader status update?

**Comment by xmonader** [comment]
in explorer devnet db i found these with workloads higher than the one mentioned in the issue @muhamadazmy what do you think?

```
49556-1 failed to apply provision: not enough free resources to support this memory size
49557-1 failed to apply provision: not enough free resources to support this memory size
```
